src/Sylius/Bundle/ApiBundle/test/fixtures/administrator.yaml

```diff
@@ -0,0 +1,8 @@
+Sylius\Component\Core\Model\AdminUser:
```

Comment:
I would say, that this file should be placed in the config folder -> `src/Sylius/Bundle/ApiBundle/test/config/fixtures/administrator.yaml`